variables.tf

```
@@ -456,3 +456,9 @@ variable "subnet_group_name" {
   type        = string
   default     = ""
 }
+
+variable "allow_traffic_inside_security_group" {
```

@kevcube sorry to do it after I've already approved, but do you mind renaming this to `intra_secruity_group_traffic_enabled` or something similar? We do have a best practice around `bool` type variables to try and name them with the `_enabled` suffix and I almost forgot about it. 

```suggestion
variable "intra_security_group_traffic_enabled" {
```